doc/changes/latest.inc

```diff
@@ -130,6 +130,7 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug with :func:`mne.io.read_raw_nihon` where latin-1 channels could not be read (:gh:`10429` :newcontrib:`Matthias Eberlein`_)
```

Comment:
You'll need to add a link to `doc/changes/names.inc` as well. If you put it in the middle somewhere (as opposed to the very bottom) you're less likely to hit a confilct